ocaml/idl/datamodel.ml

```diff
@@ -1052,7 +1052,8 @@ let _ =
     ~doc:"You attempted to start a VM that's attached to more than one VDI with a timeoffset marked as reset-on-boot." ();
   error Api_errors.vms_failed_to_cooperate [ ]
     ~doc:"The given VMs failed to release memory when instructed to do so" ();
-
+  error Api_errors.ballooning_timeout_before_migration [ ]
+    ~doc:"Timeout trying to balloon memory before VM migration. Please increase the dynamic-memory-min value." ();
```

Comment:
I'd be a little more cautious, e.g. "If the error persists, try to include the VM's memory_dynamic_min value."